packages/backend/src/models/SlackAuthenticationModel.ts

```diff
@@ -118,7 +118,8 @@ export class SlackAuthenticationModel {
             .select<(DbSlackAuthTokens & DbOrganization)[]>('*')
             .where('organization_uuid', organizationUuid);
         if (row === undefined) {
-            throw new Error(`Could not find slack installation`);
+            return row;
+            // throw new Error(`Could not find slack installation`);
```

Comment:
Adding an explicit undefined instead of 'row' so it's easier to read the code. 

```suggestion
            return undefined;
```